serveradmin/servershell/views.py

```diff
@@ -281,14 +281,25 @@ def _edit(request, server, edit_mode=False, template='edit'):  # NOQA: C901
             if server.object_id:
                 action = 'edited'
                 created = []
-                changed = [server._serialize_changes()]
+
+                changes = server._serialize_changes()
+                if len(changes.keys()) > 1:
+                    changed = [changes]
+                else:
+                    # Only object_id has been passed and we do not support
+                    # changing it (yet).
+                    changed = []
             else:
                 action = 'created'
                 created = [server]
                 changed = []
 
             try:
-                commit_obj = commit_query(created, changed, user=request.user)
+                if not created and not changed:
+                    messages.info(request, str('Nothing has changed.'))
```

Comment:
In general, this is not needed to be inside the `try`-block.

Reply:
True. Will remove it.